src/main/java/com/minecolonies/coremod/util/InventoryUtils.java

```diff
@@ -25,35 +25,6 @@
  */
 public final class InventoryUtils
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') class "InventoryUtils" has 75 methods, which is greater than the 35 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)